src/Stryker.CLI/Stryker.CLI.UnitTest/StrykerCLIInitCommandTests.cs

```diff
@@ -0,0 +1,31 @@
+using Moq;
+using Stryker.CLI.Clients;
+using Stryker.CLI.Logging;
+using Stryker.Core;
+using Xunit;
+
+namespace Stryker.CLI.UnitTest;
+
+public class StrykerCLIInitCommandTests
+{
+    private readonly StrykerCli _target;
+    private readonly Mock<IStrykerRunner> _strykerRunnerMock = new(MockBehavior.Strict);
+    private readonly Mock<IStrykerNugetFeedClient> _nugetClientMock = new(MockBehavior.Strict);
+    private readonly Mock<ILoggingInitializer> _loggingInitializerMock = new();
+
+    public StrykerCLIInitCommandTests()
+    {
+        _target = new StrykerCli(_strykerRunnerMock.Object, null, _loggingInitializerMock.Object, _nugetClientMock.Object);
+    }
+
+    [Theory]
+    [InlineData("init")]
+    [InlineData("init", "--config-file", "\"test.json\"")]
```

Comment:
Will the command create a file named `test.json` in this case?

Reply:
Yes. Currently writing integration tests to assert this 😄